adapter/src/test/java/com/twilio/kudu/sql/PaginationIT.java

```diff
@@ -73,11 +78,13 @@
   public static final String[] ACCOUNTS = new String[] { ACCOUNT1, ACCOUNT2 };
   public static final long[] TIMESTAMP_PARTITIONS = new long[] { T1, T2, T3 };
   public static final int NUM_ROWS_PER_PARTITION = 10;
+  public static KuduTable kuduTable;
 
   private final boolean descending;
   private final String tableName;
 
   private static String JDBC_URL;
+  private static String hint = "/*+ USE_OR_CLAUSE */";
```

Comment:
nit: Would `DISABLE_IN_LIST_OPTIMIZATION` be clearer?

Reply:
changed to `DISABLE_IN_LIST_OPTIMIZATION`